sdk/dnsresolver/Azure.ResourceManager.DnsResolver/tests/Tests/DnsResolverDomainListTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources.Models;
+using Azure.Core;
+using Azure.ResourceManager.DnsResolver.Models;
+
+namespace Azure.ResourceManager.DnsResolver.Tests
+{
+    public class DnsResolverDomainListTests : DnsResolverTestBase
+    {
+        private DnsResolverDomainListCollection _dnsResolverDomainListCollection;
+
+        public DnsResolverDomainListTests(bool async) : base(async)//, RecordedTestMode.Record)
+        {
+        }
+
+        public async Task CreateDnsResolverDomainListCollectionAsync()
+        {
+            var subscription = await Client.GetSubscriptions().GetAsync(TestEnvironment.SubscriptionId);
+            var resourceGroup = await CreateResourceGroupAsync();
+
+            _dnsResolverDomainListCollection = resourceGroup.GetDnsResolverDomainLists();
+        }
+
+        [Test]
+        [RecordedTest]
+        public async Task CreateDnsResolverDomainListAsync()
+        {
+            // ARRANGE
+            var dnsResolverDomainListName = Recording.GenerateAssetName("dnsResolverDomainList-");
+            await CreateDnsResolverDomainListCollectionAsync();
+            var dnsResolverDomainListData = new DnsResolverDomainListData(this.DefaultLocation, new List<string>() { "test.com.", "env.com." });
```

Comment:
It's generally preferred to use reserved (eg: example.com) or Microsoft-owned (eg: contoso.com) domains when needed.

Reply:
Fixed